### PR TITLE
fix(ui): Added bootstrap style validation on Position page on Career Site

### LINKED ISSUE #22: Career site fixes: Position page - Application form validation
**Problem**
On Career site, Position page, for Application form:

- We show validation via browser tooltip (i.e. "Please fill out this field"), while initially planned to use [bootstrap styles](https://getbootstrap.com/docs/5.3/forms/validation/#custom-styles) (as in Hub on Change password page)

**Solution**
- Try to switch to [bootstrap styles](https://getbootstrap.com/docs/5.3/forms/validation/#custom-styles) for validation (problem is that bootstrap uses js to intercept browser validation and show own instead, so we need to test that validation will look correct in case js isn't loaded)

### PR DESCRIPTION
**Pull request by Lyams**
Closes #22.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

**Comment by Lyams** [comment]
@delavaler Would we like to add related error display here also from the backend (for errors related to the relevant fields)?

**Comment by delavaler** [comment]
> @delavaler Would we like to add related error display here also from the backend (for errors related to the relevant fields)?

Yes, would be good to show errors related to fields with bootstrap style near these inputs.

**Comment by delavaler** [comment]
@Lyams  maybe I wasn't clear enough, but I expect to create all style related changes from the update to tabler branch, since tabler has own styles, for input validations as well

**Comment by Lyams** [comment]
wrong parent branch. Create PR with right parent(main) branch https://github.com/freeats/freeats/pull/137

File: app/views/career_site/positions/show.html.slim [hunk 1]
```diff
@@ -41,25 +41,34 @@
...
-          required: true,
-          type: :email,
-          pattern: EmailRegexp::HTML_EMAIL_PATTERN,
-          placeholder: "Email" \
-        )
-      = form.file_field :file, class: "form-control", id: "formFile", type: "file",
-                               required: true, accept: ".pdf,.doc,.docx,.odt,.rtf"
-
+                       class: "d-flex flex-column gap-2 needs-validation" do |form|
+      .col
```
[the remark this answers is not in the document]
Comment (Lyams) [reply app/views/career_site/positions/show.html.slim:45]: https://github.com/freeats/freeats/pull/137/files#r1838465962

File: config/locales/career_site.en.yml [hunk 2]
```diff
@@ -6,6 +6,9 @@ en:
       apply_button: Apply
       closed_alert: This position is no longer active.
       description: Description
+      invalid_email_error: The email should be valid.
```
[the remark this answers is not in the document]
Comment (Lyams) [reply config/locales/career_site.en.yml:9]: https://github.com/freeats/freeats/pull/137/files#r1838476135

File: app/assets/javascript/src/shared/bootstrap_activators.js [hunk 3]
```diff
@@ -16,3 +16,37 @@ $(document).on("click", ".toggle-chevron-content", function toggleIcons() {
   "turbo:submit-start",
   "turbo:frame-render",
 ].forEach((eventName) => document.addEventListener(eventName, () => removeTooltips()));
+
+$(document).on(
```
[the remark this answers is not in the document]
Comment (Lyams) [reply app/assets/javascript/src/shared/bootstrap_activators.js:20]: https://github.com/freeats/freeats/pull/137/files#r1838476135

**Pull request closed without merging**